Makefile

```diff
@@ -89,6 +89,10 @@ test-e2e: ## Run e2e test
 	# KUBECONFIG and SSH_PK dirs needs to be mounted inside a container if tests are run in containers
 	go test -timeout 20m -v sigs.k8s.io/cluster-api-provider-aws/test/machines -kubeconfig $${KUBECONFIG:-~/.kube/config} -ssh-key $${SSH_PK:-~/.ssh/id_rsa} -actuator-image $${ACTUATOR_IMAGE:-gcr.io/k8s-cluster-api/aws-machine-controller:0.0.1} -cluster-id $${ENVIRONMENT_ID:-""} -ginkgo.v
 
+.PHONY: validate-e2e
+validate-e2e: ## Run e2e validation/gating test
```

Comment:
Can you extend the `make check` to run it as well?

Reply:
That doesn't seem right. Currently you can run `make check` without any external dependencies but `validate-e2e` needs a running cluster to be executed.

Reply:
Oh, make sense. Ignore the comment :)

Reply:
Can we don't introduce skew unless there's reason? https://github.com/openshift/machine-api-operator/blob/master/Makefile#L48

Reply:
`test-e2e` is already taken

Reply:
I see so can we please (may be in different PR) make sure we rename the vanilla tests to test-*
So we use test-e2e consistently across all repos for the product e2e testing

Reply:
Yup, I'll do it.

Reply:
Done in https://github.com/openshift/cluster-api-provider-aws/pull/117